examples/interactions/commands/application_commands.py

```diff
@@ -73,7 +75,7 @@ async def favorite(
             )
         case 'Spy x Family':
             await inter.resp.send(
-                "I have a friend which really likes this anime, "
+                'I have a friend which really likes this anime, '
```

Comment:
Reverts out of scope change.
```suggestion
                "I have a friend which really likes this anime, "
```